### PR TITLE
Managing dns

### PR DESCRIPTION
**Pull request by MattSurabian**
Just like it says on the tin. I still kind of feel like we should avoid using terraform here just to speed things along. We can demonstrate or have folks try out the whole "dns comes along for the ride" thing when they actually have a real server setup if we have time.

### CODE REVIEW

**Comment by tkellen** [comment]
reviewin' now! I'm on board with holding off on Terraform.

**Comment by tkellen** [comment]
pushed some edits, lemme know what you think

File: exercises/managing-dns/overview.md [hunk 1]
```diff
@@ -36,3 +144,5 @@ Specify a nameserver to check using @
 - How do our local machines know which DNS server to communicate with?
 - How can we find out what DNS records exist on a domain?
 - How do you manage DNS entries as terraform resources?
```
Comment (tkellen) [inline exercises/managing-dns/overview.md:146]: should we remove this?

Comment (MattSurabian) [inline exercises/managing-dns/overview.md:146]: Probably, though maybe not. We can review the various providers on the docs page. The answer could be as simple as, "lookup your provider in the terraform docs"

**Comment by MattSurabian** [comment]
:+1: to edits

File: exercises/managing-dns/overview.md (hunk 1 above)
Comment (MattSurabian) [inline exercises/managing-dns/overview.md:146]: Yeah I think not especially since we talk about API access being essential to doing this.

**Pull request merged**